src/views/List.jsx

```diff
@@ -7,11 +7,16 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{
+					/**
+					 * TODO: write some JavaScript that renders the `data` array
+					 * using the `ListItem` component that's imported at the top
+					 * of this file.
+					 */
```

Comment:
Woohoo! This 'to-do' is now 'to-done'! Let's get rid of this comment to keep our code nice and tidy 😃 

- [ ] delete comments

Reply:
done